homeassistant/components/yolink/climate.py

```diff
@@ -11,17 +11,38 @@
     FAN_ON,
     PRESET_ECO,
     PRESET_NONE,
+    HVACAction,
     HVACMode,
 )
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.const import ATTR_TEMPERATURE, TEMP_CELSIUS
+from homeassistant.const import TEMP_CELSIUS
 from homeassistant.core import HomeAssistant, callback
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from .const import ATTR_COORDINATORS, ATTR_DEVICE_THERMOSTAT, DOMAIN
 from .coordinator import YoLinkCoordinator
 from .entity import YoLinkEntity
 
+HA_MODEL_2_YOLINK = {
+    HVACMode.COOL: "cool",
+    HVACMode.HEAT: "heat",
+    HVACMode.AUTO: "auto",
+    HVACMode.OFF: "off",
+}
+
+YOLINK_MODEL_2_HA = {
+    "cool": HVACMode.COOL,
+    "heat": HVACMode.HEAT,
+    "auto": HVACMode.AUTO,
```

Comment:
Does the yolink auto mode fall within the specs of the Home Assistant auto mode?

https://github.com/home-assistant/core/blob/3d78240ceecf3263e69eeb58484a86b30ddc23b9/homeassistant/components/climate/const.py#L23-L25

Reply:
Yes, Yolink thermostat auto mode base on a schedule.